stylus/components/forms.styl

```diff
@@ -231,8 +231,10 @@ $label
     display block
```

Comment:
Le "New styles" au dessus peut être enlevé non ?

Reply:
well there's still the old styles above that comment and some of them are still used in some apps, so I'm guessing this is still relevant.
Although I agree it would be nice to clean up things a bit and update those apps… 
You know what? I'm gonna make an issue right now. :)